bcda/auth/okta_test.go

```diff
@@ -44,9 +43,14 @@ func (s *OktaAuthPluginTestSuite) TestDeleteClient() {
 }
 
 func (s *OktaAuthPluginTestSuite) TestGenerateClientCredentials() {
-	r, err := s.o.GenerateClientCredentials([]byte("{}"))
-	assert.Nil(s.T(), r)
-	assert.Equal(s.T(), "not yet implemented", err.Error())
+	validClientID := "0oaj4590j9B5uh8rC0h7"
```

Comment:
Do we want this hard-coded twice (also in `oktaclient_test.go`)?

Reply:
only other idea i had was an env var.  do we want to go that route?

Reply:
It _feels_ more ephemeral than something we want to put in code, and it will be a different value for production.  If env vars are all we've got at the moment, maybe that's better than a surprise later on.

Reply:
well, it'll pass mokta with any string of length 20.  it's for testing against the real okta in oktaclient.go that we'll need a real clientID for

Reply:
OK, for the sake of oktaclient.go I'm guessing the test suite won't ever run against the production environment, so that issue is less concerning.  We'll try not to delete the referenced client on pain of a new PR.  😬

Reply:
once we settle on an established "testing client" in okta, i think it will make sense to keep the id as an env var

Reply:
The future solution is to create the data we need in Okta for a test and then clean it up. For example, this test could use RegisterClient() to create a client, then GenerateClientCredentials() to generate it new credentials, and then it can prove that the new credentials are, in fact, different from the original. Before we do that, though, we need a delete client function. I opened BCDA-951 for improving testing. Add comments there.